domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/service/EntityToResponseDTOMapper.java

```diff
@@ -57,6 +57,14 @@
     )
     CaseEvent map(EventEntity eventEntity);
 
+    @Mapping(
+        expression = "java(eventLiteEntity.getPreStates().isEmpty() ? java.util.Collections.emptyList()" +
```

Comment:
Aren't we missing this line from the mapping?:
`java(eventEntity.isCanCreate() ? java.util.Collections.emptyList()`
Seems that we have it for:
`    CaseEvent map(EventEntity eventEntity);`

Reply:
Thought it was not relevant at first but you're right, added the missing checks 👍